src/prefect_server/services/loop_service.py

```diff
@@ -16,35 +17,35 @@ class LoopService:
     define the `run_once` coroutine to describe the behavior of the service on each loop.
     """
 
-    # if set, and no `loop_seconds` is provided, the service will attempt to load
-    # `loop_seconds` from this config key
-    loop_seconds_config_key = None
-
-    # if no loop_seconds_config_key is provided, this will be the default
+    # The number of seconds the loop should repeat on when in operation
     loop_seconds_default = 600
 
+    # The number of seconds the loop should repeat on when testing
+    loop_seconds_debug = 1
+    debug_environment_key = "PYTEST_CURRENT_TEST"
+
     # shutdown flag for gracefully exiting the infinite loop
     is_running = True
 
-    def __init__(self, loop_seconds: Union[float, int] = None):
```

Comment:
Passing loop seconds to `__init__` is unused and is being removed for simplification.